app/components/contributor-list/component.js

```diff
@@ -4,6 +4,7 @@ import _ from 'lodash';
 export default Ember.Component.extend({
     store: Ember.inject.service(),
     session: Ember.inject.service(),
+    tagName: "section",
```

Comment:
remove an extra div

Reply:
the div this was styling was a wrapper div at the root of the component, but the component also insertis its own div. here, instead of applying the styles to the div in the template, we remove the template and instead apply the styles and attributes to the div the component provides.

Reply:
ahhh gotcha